app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java

```diff
@@ -973,7 +974,11 @@ public boolean isAutocompleteEnabled(Context context) {
 
         @Override
         public boolean isAutoFillEnabled(Context context) {
-            return SettingsStore.getInstance(context).isAutoFillEnabled();
+            // Chromium doesn't provide an autofill option.
+            // Instead, it provides a login selection dialog that let users decide whether to autofill the saved password or not.
+            // We'll only show this dialog if users enable autofill.
+            showLoginSelectDialog = SettingsStore.getInstance(context).isAutoFillEnabled();
+            return true;
```

Comment:
This is not correct. For Gecko it'll return true unconditionally even if autofill is disabled.

Reply:
In any case I think the current code is correct, we should return here what the settings have.